app/styles/tailwind/config.js

```diff
@@ -2,22 +2,116 @@
 
 module.exports = {
   theme: {
+    /* ~~ Override ~~ */
+
+    // Colors //
     colors: {
       transparent: 'transparent',
       black: '#000',
       white: '#fff',
-      grey: {
-        onyx: '#333',
-        asphalt: '#666',
-      },
+
+      // Accent Color
       blue: {
-        default: '#0068ff'
+        default: '#0068ff',
+        hover: '#0558dd',
+        medium: '#66a4ff',
+        light: '#cce1ff',
+        'light-2': '#deefff',
```

Comment:
That's a good point! Yeah I think it makes a lot of sense to change the names, and I like the idea of using numbers a lot. I've made that update, let me know if you think any of the numbering should be adjusted, or if we should get rid of the `default` keys.

I went with 'lower numbers are lighter, higher numbers are darker' numbering scheme. This is what the default Tailwind config does, and also, since our colors tend to be on the light side, this gives us more room to grow if we ever need darker colors (like if we ever did a dark mode).

Although on the other hand, 'lower numbers are darker' might feel more intuitive, so I wouldn't be opposed to reversing it.